src/main/resources/templates/addNewTask.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="pl" xmlns:th="http://www.thymeleaf.org"
+      xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+      layout:decorate="~{layout}">
+<main layout:fragment="content">
+  <form method="post" th:action="@{/add}" th:object="${newTask}">
+    <div class="mb-3">
+      <label for="taskName" class="form-label">Nazwa zadania:</label>
+      <input type="text" th:field="*{name}" id="taskName">
+    </div>
+    <div class="mb-3">
+      <label for="deadline" class="form-label">Deadline:</label>
+      <input type="date" th:field="*{deadline}" id="deadline">
+    </div>
+    <div class="mb-3">
+      <label for="category" class="form-label">Kategoria:</label>
+      <select th:field="*{category}" id="category">
+        <option th:text="|obowiazki domowe|" value="HOUSEHOLD"></option>
+        <option th:text="praca" value="WORK"></option>
+        <option th:text="szkolenie" value="TRAINING"></option>
```

Comment:
Przy enumach warto używać takiej notacji:
```
<select th:field="*{category}" id="category">
  <option th:each="cat: ${T(com.example.springDataJpa.Category).values()}"
          th:text="${cat.description}" th:value="${cat.name()}">
    Przykładowa kategoria
  </option>
</select>
```
dzięki temu jeżeli dodasz, albo usuniesz coś z enuma, to nie musisz pamiętać o aktualizacji szablonów